scipy/sparse/linalg/dsolve/meson.build

```diff
@@ -0,0 +1,189 @@
+subdir('tests')
```

Comment:
can this go at the end? better to keep to that pattern - also good for ordering, build the main dir before going into any subdirs

Reply:
Sure.